P/PicoSAT/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PicoSAT"
+version = v"965.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://fmv.jku.at/picosat/picosat-965.tar.gz", "15169b4f28ba8f628f353f6f75a100845cdef4a2244f101a02b6e5a26e46a754")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/picosat*
+if [[ ${target} == *musl* ]]; then
+    sed -i 's!sys/unistd.h!unistd.h!g' picosat.c
+elif [[ ${target} == *mingw* ]]; then
+    sed -i 's!case X"$CC" in!case X"$target" in!g' configure.sh
+fi
+if [[ "${nbits}" == 32 ]]; then
+    ./configure.sh --shared --32
+else
+    ./configure.sh --shared
+fi
```

Comment:
As far as I can tell from the script, the only thing that `--32` does it to set
```
CFLAGS="$CFLAGS -m32"
```
which is pretty much useless for us since we already use the relevant compiler
```suggestion
./configure.sh --shared
```